core/src/commonMain/kotlin/com/ustadmobile/core/viewmodel/clazz/detailoverview/ClazzDetailOverviewViewModel.kt

```diff
@@ -275,9 +286,11 @@ class ClazzDetailOverviewViewModel(
         }
     }
 
-    private fun onClickEdit() {
-        navController.navigate(ClazzEditViewModel.DEST_NAME,
-            mapOf(UstadView.ARG_ENTITY_UID to entityUidArg.toString()))
+    fun onClickEdit(actionType: String) {
```

Comment:
NO!

For two reasons:
1) This is clearly against the ViewModel event pattern : for any event, there should be a function onClickEdit means the user clicked edit. You should add a separate onClickCopy function.
2) Even if that was going to be done, a string parameter is the wrong way to handle that. A string can have an infinite number of values. In this case there are only two valid values (so if one was doing it that way, one should use a boolean or enum)